packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/TxDetailModal/AdvancedTxDetails/IODetails/IODetails.tsx

```diff
@@ -150,12 +150,14 @@ interface IOGridRow {
     anonymitySet?: AnonymitySet;
     tx: WalletAccountTransaction;
     vinvout: WalletAccountTransaction['details']['vin'][number];
+    isPhishingTransaction?: boolean;
```

Comment:
This is a false-positive field, ESlint is claiming that it is not used. You can add `// eslint-disable-next-line react/no-unused-prop-types` to solve it.

Reply:
I added it to `CollapsibleIOSection` which extends the `IOSectionColumnProps` type

Reply:
Then it would be nice to refactor the types a bit so the `type-check_and_lint` passes